examples/seismic/tti/tti_example.py

```diff
@@ -37,12 +37,14 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
     # Define receiver geometry (spread across `x, y`` just below surface)
     rec, u, v, summary = solver.forward(save=save, autotune=autotune)
 
-    if preset == 'constant':
+    if preset == 'constant-tti':
         # With  a new m as Constant
         v0 = Constant(name="v", value=2.0, dtype=np.float32)
-        solver.forward(save=save, vp=v0)
+        solver.model.update('vp', v0.value)
```

Comment:
this shouldn't be required strictly speaking?

Reply:
I prefer the runtime argument `vp=v0` to avoid keeping track of what was changed in the model

Reply:
Well, `v0` is a constant and passing it directly to `solver.forward` won't work. It gives the following error:
```
Traceback (most recent call last):
  File "devito/examples/seismic/tti/tti_example.py", line 103, in <module>
    run(shape=shape, spacing=spacing, nbl=args.nbl, tn=tn,
  File "devito/examples/seismic/tti/tti_example.py", line 48, in run
    solver.forward(vp=2., save=save)
  File "/home/oscar/devito/examples/seismic/tti/wavesolver.py", line 155, in forward
    summary = self.op_fwd(save).apply(src=src, rec=rec, u=u, v=v,
  File "/home/oscar/devito/devito/operator/operator.py", line 708, in apply
    args = self.arguments(**kwargs)
  File "/home/oscar/devito/devito/operator/operator.py", line 557, in arguments
    args = self._prepare_arguments(**kwargs)
  File "/home/oscar/devito/devito/operator/operator.py", line 449, in _prepare_arguments
    args.update(p._arg_values(**kwargs))
  File "/home/oscar/devito/devito/types/dense.py", line 843, in _arg_values
    for i, s in zip(self.dimensions, new.shape):
AttributeError: 'float' object has no attribute 'shape'
```


and for that reason I used `model.update`